src/js/tracing/integrations/nativeFrames.ts

```diff
@@ -1,239 +1,201 @@
-import { spanToJSON } from '@sentry/core';
 import type { Client, Event, Integration, Measurements, MeasurementUnit, Span } from '@sentry/types';
 import { logger, timestampInSeconds } from '@sentry/utils';
 
 import type { NativeFramesResponse } from '../../NativeRNSentry';
-import type { ReactNativeClientOptions } from '../../options';
+import { AsyncExpiringMap } from '../../utils/AsyncExpiringMap';
 import { isRootSpan } from '../../utils/span';
 import { NATIVE } from '../../wrapper';
 
+/**
+ * Timeout from the start of a span to fetching the associated native frames.
+ */
+const FETCH_FRAMES_TIMEOUT_MS = 2_000;
+
+/**
+ * Timeout from the final native frames fetch to processing the associated transaction.
+ * If the transaction is not processed by this time, the native frames will be dropped
+ * and not added to the event.
+ */
+const END_FRAMES_TIMEOUT_MS = 2_000;
```

Comment:
I can not really see the difference of `END_FRAMES_TIMEOUT_MS` and `START_FRAMES_TIMEOUT_MS` by the comments since they are exactly the same. Does the Start frame apply to the start of the native frames fetch?